lib/webenginesettings.cpp

```diff
@@ -0,0 +1,122 @@
+/****************************************************************************
+**
+** Copyright (C) 2016 Jolla Ltd.
+** Contact: Raine Makelainen <raine.makelaine@jolla.com>
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "webenginesettings.h"
+
+#include <silicatheme.h>
+
+#include <QtCore/QLocale>
+#include <QtCore/QSettings>
+#include <QtCore/QSize>
+#include <QtCore/QtMath>
+#include <QtGui/QGuiApplication>
+#include <QtGui/QScreen>
+
+Q_GLOBAL_STATIC(SailfishOS::WebEngineSettings, webEngineSettingsInstance)
+
+#define SAILFISH_WEBENGINE_DEFAULT_PIXEL_RATIO 1.5
+
+static bool testScreenDimensions(qreal pixelRatio) {
+    QScreen *screen = QGuiApplication::primaryScreen();
+    qreal w = screen->size().width() / pixelRatio;
+    qreal h = screen->size().height() / pixelRatio;
+    return fmod(w, 1.0) == 0 && fmod(h, 1.0) == 0;
+}
+
+const QSettings &quickSettings()
+{
+    static const QSettings settings(QSettings::SystemScope, QStringLiteral("QtProject"), QStringLiteral("QtQuick2"));
```

Comment:
Why like this?  I would have expected UserScope with an applicationName derived from the application binary path (or read from /proc/pid etc)